<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StructuraSystems::Client::MarkdownElement</class>
 <widget class="QWidget" name="StructuraSystems::Client::MarkdownElement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>394</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTextBrowser" name="TextBrowser">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::ContextMenuPolicy::CustomContextMenu</enum>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::SizeAdjustPolicy::AdjustToContents</enum>
       </property>
       <property name="tabChangesFocus">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="openExternalLinks">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="TextEditor">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::ContextMenuPolicy::CustomContextMenu</enum>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::SizeAdjustPolicy::AdjustToContents</enum>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="LanguageCombobox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>Markdown</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>YaML</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SysMLv2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>KerML</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QToolButton" name="MoveElementUp">
       <property name="styleSheet">
        <string notr="true">QToolButton {
border: none;
background: transparent;
}

QToolButton:hover {
border: 1px solid blue;
background: red;
}</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="MoveElementDown">
       <property name="styleSheet">
        <string notr="true">QToolButton {
border: none;
background: transparent;
}

QToolButton:hover {
border: 1px solid blue;
background: red;
}</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionAddElementAbove">
   <property name="text">
    <string>AddElementAbove</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionAdd_Element_Below">
   <property name="text">
    <string>Add Element Below</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionEdit">
   <property name="text">
    <string>Edit</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
